llvm/test/DebugInfo/codeview-bit-slice-fragments.ll

```diff
@@ -0,0 +1,67 @@
+; RUN: llc -filetype asm -o - %s | FileCheck %s
+
+; Ensure that we do not emit any live ranges for the fragment as it is
+; bit-sliced which cannot be represented in CodeView.
+; CHECK-NOT: .cv_def_range
+
+source_filename = "/tmp/reduced.ll"
+target datalayout = "e-m:w-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-unknown-windows-msvc19.37.32825"
+
+define swifttailcc void @"$s12SourceKitLSP0aB6ServerC14workspaceTestsySay08LanguageD8Protocol19WorkspaceSymbolItemOGSgAE0iF7RequestVYaKFTY0_"() !dbg !5 {
+entryresume.0:
+  br label %.from.63
+
+.from.63:                                         ; preds = %.from.63, %entryresume.0
+  %lsr.iv = phi ptr [ %scevgep, %.from.63 ], [ null, %entryresume.0 ]
+  %0 = load i64, ptr %lsr.iv, align 8
+  %1 = load i64, ptr %lsr.iv, align 8
+  call void @llvm.dbg.value(metadata ptr %lsr.iv, metadata !9, metadata !DIExpression(DW_OP_deref, DW_OP_LLVM_fragment, 1, 1)), !dbg !22
+  call void @llvm.dbg.value(metadata ptr %lsr.iv, metadata !9, metadata !DIExpression(DW_OP_deref, DW_OP_LLVM_fragment, 2, 64)), !dbg !22
+  %2 = load volatile ptr, ptr null, align 8, !dbg !30
+  %scevgep = getelementptr i8, ptr %lsr.iv, i64 8
+  br label %.from.63
+}
+
+declare void @llvm.dbg.value(metadata %0, metadata %1, metadata %2) #0
+
+attributes #0 = { nocallback nofree nosync nounwind speculatable willreturn memory(none) }
```

Comment:
I am getting consistent failures in this branch because memory(none) is not supported in 5.10. The memory(…) attributes introduced in https://reviews.llvm.org/D135780 which are not part of the release/5.10 branch. You might be able to rewrite the test without memory(none) and get the same result. See https://github.com/apple/llvm-project/pull/7484 for a similar problem that was solved removing the memory(…) attributes.

Reply:
Thanks for the heads up @drodriguez!  I wonder why the CI didn't catch that.  I'll put up a PR to drop the attribute.

Reply:
I think one needs to be explicit with "please test llvm" even in the LLVM repo. But it has not been successful for a while, so… (see https://ci.swift.org/job/pr-apple-llvm-project-llvm-macos/ and https://ci.swift.org/job/pr-apple-llvm-project-llvm-linux/)